monai/tests/test_clinical_preprocessing.py

```
@@ -0,0 +1,144 @@
+    except ValueError as e:
+        error_message = str(e)
+        assert "Unsupported modality" in error_message
+        assert "PET" in error_message
+        assert "CT, MR, MRI" in error_message
+
+
+def test_preprocess_dicom_series_invalid_type():
+    """Test preprocess_dicom_series raises TypeError for non-string modality."""
+    try:
+        preprocess_dicom_series("dummy_path.dcm", 123)
+        assert False, "Should have raised TypeError"
```

Ruff (0.14.8)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)